### ISSUE TITLE
Training the diffusion prior

### ISSUE THREAD

**Issue by bluestyle97** [NasirKhalid24/CLIP-Mesh#5]
Hi, congratulations on your fantastic work! Could you please share the code for training the diffusion prior? I want to be more clear about some training details. Besides, I'm curious if the CLIP text embeddings and image embeddings  are normalized when training the diffusion prior network. Thanks!

**Comment by NasirKhalid24** [comment]
Hey - the training script is from the [following repo](https://github.com/lucidrains/DALLE2-pytorch/tree/2eac7996faef08803230c7f3b2f9199e07c33b39), the embeddings used are [LAION precomputed](https://laion.ai/blog/laion-400-open-dataset/) ones
